### PR TITLE
Modify 1-click to use the demo-script for releasing products and setting up demos

### PR DESCRIPTION
**Pull request by Kinshuk1993**
Modify 1-click to use the demo-script for releasing products and setting up demos:

- Create a sample JSON file with replaceable vars/strings
- Call demo script with above JSON file with required storage, apic params and demos
- Test 1-click E2E
  -  All demos
  - EEI
  - DDD

### CODE REVIEW

File: products/bash/1-click-install.sh [hunk 1]
```diff
@@ -236,66 +260,72 @@ fi
...
 else
-  echo "INFO: Creating $JOB_NAMESPACE namespace"
+  echo -e "$INFO [INFO] Creating the '$JOB_NAMESPACE' namespace"
   if ! oc create namespace $JOB_NAMESPACE; then
-    echo -e "$cross ERROR: Failed to create the $JOB_NAMESPACE namespace" 1>&2
+    echo -e "$CROSS [ERROR] Failed to create the '$JOB_NAMESPACE' namespace"
     divider
     exit 1
+  else
+    echo -e "$TICK [SUCCESS] Succesfully created the '$JOB_NAMESPACE' namespace"
```
Comment (James-Kirk) [inline products/bash/1-click-install.sh:271]: Successfully*

Comment (Kinshuk1993) [reply products/bash/1-click-install.sh]: Done, thanks

**Review by James-Kirk — APPROVED** [review]
Just on typo to fix, otherwise LGTM :)

**Review by hrizvi1 — APPROVED** [review]
LGTM

**Review by DanRoseus — APPROVED** [review]
Looks good

**Pull request merged**